app/controllers/overrides/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Overrides
+  class OmniauthCallbacksController <
+    DeviseTokenAuth::OmniauthCallbacksController
+    def assign_provider_attrs(user, auth_hash)
```

Comment:
I think this needs comments or needs to be simplified. Could you walk me through what it does?

Reply:
I've linked to the wiki suggesting this method logic.

Reply:
That's helpful. Could you add another one-liner explaining what it does exactly?